src/filtrering/filtrering-veileder-grupper/ny_veiledergrupper-innhold.tsx

```diff
@@ -115,36 +91,4 @@ function NyVeilederGruppeInnhold(props: VeilederGruppeInnholdProps) {
     );
 }
 
-interface VeilederGruppeRad {
-    hanterVelgGruppe: (e: React.ChangeEvent<HTMLInputElement>) => void;
-    veilederGruppe: VeiledergrupperFilter;
-    veiledereFilter: string[];
-    onClickRedigerKnapp: () => void;
-}
-
-function VeilederGruppeRad({veilederGruppe, hanterVelgGruppe, onClickRedigerKnapp, veiledereFilter}: VeilederGruppeRad) {
```

Comment:
Ah, if you deleted this and combined MineFilterRad and this, you don't have to rename LagretFilterRad that I commented above!!

Reply:
No no, ny_veiledergrupper-innhold.tsx is still present...I didn't consider to combine those two...even though its maybe possible.